packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -222,6 +226,107 @@ export const tryEvaluateExpression = (
   return fallbackNode;
 };
 
+interface PartialBindingWithMeta {
+  node: t.Node;
+  path: NodePath;
+  constant: boolean;
+  meta: Metadata;
+}
+
+/**
+ * Will return the `node` of the a binding.
+ * This function will follow import specifiers to return the actual `node`.
+ *
+ * When wanting to do futher traversal on the resulting `node` make sure to use the output `meta` as well.
+ * The `meta` will be for the resulting file it was found in.
+ *
+ * @param path
+ */
+export const resolveBindingNode = (
+  binding: Binding | undefined,
+  meta: Metadata
+): PartialBindingWithMeta | undefined => {
+  if (binding && t.isVariableDeclarator(binding.path.node)) {
+    return {
+      node: binding.path.node.init as t.Node,
+      path: binding.path,
+      constant: binding.constant,
+      meta,
+    };
+  }
+
+  if (binding && binding.path.parentPath.isImportDeclaration()) {
+    const moduleImportName = binding.path.parentPath.node.source.value;
+    const isRelative = moduleImportName.startsWith('.');
```

Comment:
What happens if file starts with `.` 🤔? Can we use something like this https://nodejs.org/api/path.html#path_path_isabsolute_path